.github/workflows/github-actions.yml

```diff
@@ -2,9 +2,9 @@ name: Setup Builds and Tests
 
```

Comment:
Think for this file, maybe it's not can put the linting fixes for the yml file in #580

Reply:
So that we just focus on adding a new workflow for this PR #575 ba